apps/events-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx

```diff
@@ -98,7 +98,7 @@ const EventKeywords: React.FC<Props> = ({
         />
       )}
       {showIsFree && freeEvent && (
-        <Keyword
+        <KeywordButton
```

Comment:
We should think about the naming, because the spec is that it might be not even a "button". I would say its clickable or non clickable tag, and also, we have to consider to replace it with hcrc tag component.

Reply:
Perhaps I could rename it to `KeywordTag`